tsup.config.js

```diff
@@ -4,7 +4,7 @@ const shared = {
     external: ['discord.js', 'iti'],
     platform: 'node',
     clean: true,
-    sourcemap: false,
+    sourcemap: true,
```

Comment:
Why?

Reply:
we're a dev framework, it would be better if we had sourcemaps so users can debug their code. Its up to the end product to minify or not